cityofaustin/atd-moped
# Add `is_deleted` column for moped users

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9553

## Testing
**URL to test:** 
_This needs to be tested on moped-test._

**Steps to test:**
_I'm using JD's account on purpose so that we don't mess up our own Moped accounts. Sergio's account is also good for testing. You might see Cognito errors return from the Moped API for other test accounts since the staging user pool data is not consistent with the `moped_users` table in the moped-test DB._

_Moped-test uses staging's DynamoDB table for roles and Cognito user pool so you could desync those data sources from the staging DB if you modify your own account - so be careful!_

1. Go to https://moped-test.austinmobility.io/moped/staff/edit/1
2. Click **Inactivate User** to inactivate JD.
3. You should be redirected to the user table and no longer see JD's account listed in the table.
4. Try to add JD to a new or existing project team and you should not see him in the personnel list
5. Go back to the Staff table and toggle the **Show Inactive Accounts** switch to show JD in the list.
6. Click the pencil in JD's row, add a password to the edit form, and then click **Activate User** (you will get a warning if the password isn't complex enough)
7. Try to add JD to a new or existing project team and you should see him listed in the personnel list

---
#### Ship list
